### PR TITLE
Port Stream Settings to React

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

**Comment by trevorrl** [comment]
@holiber I am unable to save additional custom multistream destinations. Clicking Add will show the fields, but clicking Save does nothing. 

`TypeError: Cannot assign to read only property 'url' of object '#<Object>'`

**Comment by holiber** [comment]
@trevorrl  should be fixed now

File: app/components-react/windows/settings/Stream.tsx [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+
+/**
+ * Renders a Platform placeholder
+ */
+function Platform(p: { platform: TPlatform }) {
+  const platform = p.platform;
+  const { UserService, StreamingService } = Services;
+  const { canEditSettings, platformMerge, platformUnlink } = useStreamSettings();
+  const isMerged = StreamingService.views.checkPlatformLinked(platform);
+  const username = UserService.state.auth!.platforms[platform]?.username;
```
Comment (andycreeth) [inline app/components-react/windows/settings/Stream.tsx:284]: You're accessing `isMerged` and `username` in a non-reactive way, i.e. not using `useVuex`.  Was that intentional?

**Review by andycreeth — APPROVED** [review]

File: app/components-react/windows/settings/Stream.tsx (hunk 1 above)
Comment (holiber) [reply app/components-react/windows/settings/Stream.tsx:284]: They don't need to be reactive here

**Pull request merged**